### PR TITLE
NFV: Handle custom OVS for BZ#2228643

### PR DESCRIPTION
**Pull request by EmilienM**
To address https://bugzilla.redhat.com/show_bug.cgi?id=2228643

### CODE REVIEW

File: configs/nfv.yaml [hunk 1]
```diff
@@ -123,3 +123,5 @@ post_install: |
   sudo podman create --net=host --name=squid --volume /home/stack/squid/squid.conf:/etc/squid/squid.conf:z --volume /home/stack/squid/htpasswd:/etc/squid/htpasswd:z quay.io/emilien/squid:latest
   sudo podman generate systemd --name squid | sudo tee -a /etc/systemd/system/container-squid.service
   sudo systemctl enable --now container-squid
+  cd ~/ovs-dpdk-rpms && sudo dnf install -y *.rpm
+  ( sleep 60 ; sudo reboot ) &
```
Comment (EmilienM) [inline configs/nfv.yaml:127]: yeah a reboot is hardcore but at least I'm sure OVS will be restarted and all networking fine with DPDK.

**Review by mandre — COMMENTED** [review]
How do we track the inclusion of the fix for BZ#2228643, so that we stop installing our custom ovs package once it's in the official repos?

**Comment by MaysaMacedo** [comment]
/lgtm

@mandre https://issues.redhat.com/browse/OSASINFRA-3269

**Comment by MaysaMacedo** [comment]
/hold
to see if my @mandre is fine with tracking that on JIRA

**Review by mandre — COMMENTED** [review]
/lgtm

**Comment by EmilienM** [comment]
/hold cancel

ship it

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**